assets/patches/sidecars/node_driver_registrar.yaml

```diff
@@ -0,0 +1,28 @@
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-node-driver-registrar
+          securityContext:
+            privileged: true
+          image: ${NODE_DRIVER_REGISTRAR_IMAGE}
+          imagePullPolicy: IfNotPresent
+          args:
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/${DRIVER_NAME}/csi.sock
```

Comment:
I think `DRIVER_NAME` is too generic name for this specific purpose. Can we use `DRIVER_REG_SOCK_PATH` instead?

Reply:
But it is really the driver name. It's only used as part of two socket paths, so one `DRIVER_REG_SOCK_PATH` would not work either.  There is `/var/lib/kubelet/plugins/*/csi.sock` (driver's socket) and `/var/lib/kubelet/plugins_registry/*-reg.sock` (created by node-driver-registrar).

Reply:
Yes, I agree (I missed that this name came from `assets/csidriver.yaml`). I still think that `CSI_DRIVER_NAME` would be better because it matches to `kind: CSIDriver` in csidriver.yaml, but now it's not so important -- it's up to you.

Reply:
Btw, if `DRIVER_NAME` is first-class citizen (defined in aws-ebs.go along with `AssetPrefix` and `AssetShortPrefix`), it's not nice that magic string `"ebs.csi.aws.com"` is duplicated many times in other yamls:
```
assets/base/node.yaml:41:            path: /var/lib/kubelet/plugins/ebs.csi.aws.com/
assets/drivers/aws-ebs/base/csidriver.yaml:4:  name: ebs.csi.aws.com
assets/drivers/aws-ebs/base/storageclass_gp2.yaml:8:provisioner: ebs.csi.aws.com
assets/drivers/aws-ebs/base/storageclass_gp3.yaml:10:provisioner: ebs.csi.aws.com
assets/drivers/aws-ebs/base/volumesnapshotclass.yaml:7:driver: ebs.csi.aws.com
```
Can we use `${DRIVER_NAME}` everywhere instead of `"ebs.csi.aws.com"` ?

Reply:
I fixed `assets/base/node.yaml`, because it's the base of all daemonsets and the driver name must be replaced there.

Files in `assets/drivers/aws-ebs` are already aws-ebs specific, I think `${DRIVER_NAME}` does not make them better or more readable.